### DISCUSSION TITLE
Feature: Show if a model is updatable

### DISCUSSION THREAD

**Discussion by lirc571** [open-webui/open-webui#3758] [General]
**Is your feature request related to a problem? Please describe.**
Ollama allows updating of an existing tagged model (use an existing tag on a new model). This info is not visible in the WebUI right now.

E.g. previously `gemma2:9b-instruct-q6_K` had an ID of `37e9e747b764`, but after the recent update, the same tag has become `3827bab50424`.

**Describe the solution you'd like**
It would be great to be able to see if a model is updatable (a local model is updatable if the local hash differs from the remote hash).

**Describe alternatives you've considered**
It is hard to implement this feature directly in the current UI because ollama models are only listed in dropdowns right now. Perhaps it would be better to create a dedicated "models page" that displays a table of local models, and then implement this feature.

**Additional context**
NA

**Comment by justinh-rahb** [comment]
Absent Ollama developing and providing an API for their available models list, there's no way for us to do such a lookup for the remote hash to compare.

Related:
- #1260

**Reply by lirc571** [reply]
Well technically it's possible to get the remote hash by parsing the HTML of the model page on <https://ollama.com>. 😄 

E.g., do a `$('#file-explorer > section > div > div > div > p:nth-child(3)')` on <https://ollama.com/library/phi3:14b>.
